cpp/test/Ice/background/AllTests.cpp

```diff
@@ -132,8 +132,7 @@ allTests(TestHelper* helper)
         backgroundController->resumeCall("findAdapterById");
 
         locator = LocatorPrx(communicator, "locator:" + endp);
-        obj = obj->ice_locator(locator);
-        obj->ice_ping();
+        locator->ice_ping();
```

Comment:
It's a different test.

Before: we were pinging an indirect proxy (I assume), resolved via locator
After: we ping the locator itself

Is this undoing an accidental change from a previous PR?

Reply:
> Is this undoing an accidental change from a previous PR?

Yes, see comments in #1893. In the original version before the proxy usage refactoring in the test we ping the locator not the indirect the proxy.